nixos/lib/systemd-unit-options.nix

```diff
@@ -48,6 +48,18 @@ in rec {
       '';
     };
 
+    representation = mkOption {
```

Comment:
What about `useDropin` or `asDropin` instead? I'd probably not understand what this is doing, just by looking at the configuration name.

Reply:
Thanks, I used now asDropin and also renamed auto